### PR TITLE
Gradle update

### PR DESCRIPTION
**Pull request by manjunathc23**
Little clean up's and gradle update.

### CODE REVIEW

File: dependencies.gradle [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+        zetaUtilsVersion                      : "0.4",
+        zetaEitherVersion                     : "0.3",
+        zetaListItemDividerVersion            : "0.2",
+        zetaHeaderAndFooterAdaptersVersion    : "0.2",
+
+        stickyScrollViewVersion               : "1.0.2",
+
+        metagalacticExpandableImageViewVersion: "0.0.2",
+
+        lottieAnimationVersion                : "1.5.1",
```
Comment (manijshrestha) [inline dependencies.gradle]: 👍

File: dependencies.gradle [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+        fabricPluginVersion                   : "1.22.1",
+        gradlePluginVersion                   : "3.0.0-alpha6",
+        frodoPluginVersion                    : "0.8.3",
+        gmsServicesPluginVersion              : "3.0.0",
+
+        powerMockitoVersion                   : '1.6.2',
+]
+
+ext.gradlePlugins = [
+        fabricPlugin      : "io.fabric.tools:gradle:$versions.fabricPluginVersion",
```
Comment (manijshrestha) [inline dependencies.gradle]: I suggest we only use the one that are truly used to to keep in simple.

File: zetautils/build.gradle [hunk 3]
```diff
@@ -1,34 +1,90 @@
...
+            annotationProcessorOptions {
+                includeCompileClasspath false
+            }
+        }
     }
+
     buildTypes {
+
+        debug {
+            minifyEnabled true
```
Comment (manijshrestha) [inline zetautils/build.gradle]: this is intentional?, minify debug? Do note, release is non minified here.

**Review by manijshrestha — CHANGES_REQUESTED** [review]
Some nit comments.

File: dependencies.gradle (hunk 2 above)
Comment (manjunathc23) [reply dependencies.gradle]: Sounds good. Will remove cmd c + cmd v changes.

File: zetautils/build.gradle (hunk 3 above)
Comment (manjunathc23) [reply zetautils/build.gradle]: Oops it should be other way round.

File: dependencies.gradle (hunk 1 above)
Comment (manjunathc23) [reply dependencies.gradle]: Cleaning this up.

**Comment by manjunathc23** [comment]
@manijshrestha  : Addressed the code reviews, please take a look now.

**Review by manijshrestha — APPROVED** [review]

**Pull request merged**